src/components/Popover/Popover.vue

```diff
@@ -74,6 +74,32 @@ export default {
 	components: {
 		VPopover,
 	},
+	props: {
+		open: {
+			type: Boolean,
+			default: false,
+		},
+	},
+	data() {
+		return {
+			doShowMenu: false,
+		}
+	},
+
+	watch: {
+		open(newValue) {
+			if (newValue === true) {
+				// show with delay to allow position calculation to happen for dynamic contents,
+				// otherwise the popover will appear in the wrong place first and be visible,
+				// and then in the next tick it repositions itself
+				window.setTimeout(() => {
```

Comment:
https://vuejs.org/v2/api/#vm-nextTick?

Reply:
nope, sadly next tick happens too early...

Reply:
I'll add this in the comment so the next dev won't be wondering